resources/generate_book.py

```diff
@@ -869,6 +869,10 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
             }) for i in range(8)]),
             text(f'{detail_crop("potato")}Potatoes are a single block crop. Potato seeds can be planted on farmland to be grown, and will produce $(thing)Potatoes$() and $(thing)Potato Seeds$() as a product.', title='Potatoes').link('tfc:seeds/potato').link('tfc:food/potato').anchor('potato'),
             multimultiblock('', *[two_tall_block_spotlight('', '', 'tfc:farmland/loam', 'tfc:crop/potato[age=%d]' % i) for i in range(7)]),
+            text(f'{detail_crop("pumpkin")}Pumpkins are a spreading crop. Pumpkin seeds can be planted on farmland to be grown, and will place up to two $(thing)Pumpkin Blocks$() on the ground next to it while it is mature. If the pumpkin blocks are harvested, if the pumpkin plant matures again, it can grow more pumpkins.').link('tfc:seeds/pumpkin').anchor('pumpkin'),
+            multimultiblock('', *[two_tall_block_spotlight('', '', 'tfc:farmland/loam', 'tfc:crop/pumpkin[age=%d]' % i) for i in range(8)]),
+            text(f'{detail_crop("melon")}Melons are a spreading crop. Melon seeds can be planted on farmland to be grown, and will place up to two $(thing)Melon Blocks$() on the ground next to it while it is mature. If the melon blocks are harvested, if the pumpkin plant matures again, it can grow more melon.').link('tfc:seeds/melon').anchor('melon'),
+            multimultiblock('', *[two_tall_block_spotlight('', '', 'tfc:farmland/loam', 'tfc:crop/pumpkin[age=%d]' % i) for i in range(8)]),
```

Comment:
Maybe make these actually show the pumpkin and melon blocks getting placed on adjacent blocks? Have a 3x3 wide footprint?